guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,14 +4,39 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== New All Hosts index page improvements ===
+* On the experimental new All Hosts index page, you can now click 'Manage Columns' and add Katello-provided columns, including
+** RHEL Lifecycle status
+** Installable updates
+** Last seen
+** Lifecycle environment
+** Content view
+** Content source
+** Registered at
+* Also, you can now change the content view and lifecycle environment of multiple hosts. Select some hosts, click the vertical ellipsis, then click 'Change content view environments.'
+
+=== New manifest expiration warnings ===
+* Users will now be notified before their subscription manifest expires. The number of days of notice is configured by the expire_soon_days setting.
+* If a manifest is expired or expiring soon, you'll see new alert banners on the Manage Manifest screen.
+* The expiration date of the manifest is now displayed on the Manage Manifest screen.
+* Refreshing your manifest will now extend the expiration date of the manifest to 1 year from the current date.
+
+=== Repair content view versions ===
+* New actions have been introduced to facilitate repairs on the CV version repositories. These actions involve identifying and fixing missing or corrupted content units within the published repository. The action is available on hammer as "verify-checksum" sub-command for content-view version command.
+
+=== Content View Publish ===
+* Users will not be allowed to publish a content view if a child repository is syncing and vice-versa to avoid any data integrity issues.
+
+=== Other enhancements ===
+* Subscription expiration information has now moved from the Subscription - Entitlement report to the Subscription - General report. This report now accepts the expiring_in_days input, so you can see when your subscriptions will expire.
+* The activation keys selection on the Hostgroup Edit page has been refactored from jquery to React, providing a more modern look and user experience.
+* Katello now indexes metadata for container manifests, including annotations, labels, and identifiers for Flatpak and bootable images.
+* Container Gateway database migrated from SQLite to PostgreSQL for increased smart proxy container registry performance.
 
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
-
-There are no upgrade warnings with Katello {KatelloVersion}.
+* This release includes a migration that will run during upgrades to fetch this metadata for existing container manifests in Katello.
```

Comment:
Users have to trigger this via `foreman-maintain advanced procedure run pulpcore-container-handle-image-metadata`. So:

```suggestion
* Container content users will want to run 'foreman-maintain advanced procedure run pulpcore-container-handle-image-metadata' to pre-migrate data to avoid a lengthy migration in the future. There will be multiple releases to allow this before it is mandatory.
```